xml/rmt_install.xml

```diff
@@ -219,638 +220,721 @@
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>: <literal>[]</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: List of products to
+        enable for mirroring
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: app.scc.products_disable</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: list
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>: <literal>[]</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: List of products to
+        disabled for mirroring.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: ingress.enabled</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: bool
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>: <literal>false</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: Ingress Enabled
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: ingress.hosts[0]</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: object
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>:
+        <literal>{"host":"chart-example.local","paths":[{"path":"/","pathType":"Prefix"}]}</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: DNS name at which the
+        &rmt; service will be accessible from clients
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: ingress.tls[0].hosts[0]</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: string
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>:
+        <literal>"chart-example.local"</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: DNS name at which the
+        &rmt; service will be accessible from clients
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term><emphasis role="strong">Key</emphasis>: ingress.tls[0].secretName</term>
+      <listitem>
+       <para>
+        <emphasis role="strong">Type</emphasis>: string
+       </para>
+       <para>
+        <emphasis role="strong">Default</emphasis>:
+        <literal>"rmt-cert"</literal>
+       </para>
+       <para>
+        <emphasis role="strong">Description</emphasis>: TLS Ingress
+        Certificate.
+       </para>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+   </sect3>
+   <sect3 xml:id="sec-rmt-helm-optional-values">
```

Comment:
I propose we remove this section. It makes the document extremely long, the options are already documented in values.yaml, and most customers don't need to touch these.